appium/tests/specs/live/update/CheckAppAfterUpdateFromOldVersion.spec.ts

```diff
@@ -34,6 +34,8 @@ describe('UPDATE: ', () => {
     await driver.terminateApp(bundleId);
     //remove current app version
     await driver.removeApp(bundleId);
+    //reset keychain
+    await driver.execute('mobile: clearKeychains')
```

Comment:
This should not be part of tests if it doesn't happen in prod

Reply:
The problem here is because other tests using new build then we have `encryption key` stored in Keychain.

When we try to migrate from `UserDefaults` in old build to `Keychain` in new build - it fails because `Keychain` already has  `encryption key` from previous tests.

This situation shouldn't happen in prod as users won't have `encryption key` stored in `Keychain` when migrating from `UserDefaults`.
I can add handle for this situation - in case if decrypting with `Keychain` key fails then app will check if `UserDefaults` has encryption key and try to decrypt with it. Do we need such additional check?

Reply:
I see - I'll have a second look

Reply:
You are right, it looks good as it is. 👍 I misunderstood.